### ISSUE TITLE
Cannot validate argument on parameter 'Path'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.

### ISSUE THREAD

**Issue by fcallejon** [brunolm/nvm#1]
I've just installed the module and running `nvm install latest` throws:

```
Expand-Archive : Cannot validate argument on parameter 'Path'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
At C:\Program Files\WindowsPowerShell\Modules\power-nvm\0.3.0\power-nvm.psm1:176 char:24
+         Expand-Archive $zip -DestinationPath "$versionsDir" -Force
+                        ~~~~
    + CategoryInfo          : InvalidData: (:) [Expand-Archive], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Expand-Archive
 
Cannot index into a null array.
At C:\Program Files\WindowsPowerShell\Modules\power-nvm\0.3.0\power-nvm.psm1:181 char:5
+     $targetDir = (Join-Path $folder[0].Parent.FullName $versionToInst ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : NullArray
 
Test-Path : Cannot bind argument to parameter 'Path' because it is null.
At C:\Program Files\WindowsPowerShell\Modules\power-nvm\0.3.0\power-nvm.psm1:183 char:19
+     if (Test-Path $targetDir) {
+                   ~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Test-Path], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Microsoft.PowerShell.Commands.TestPathCommand
 
Cannot index into a null array.
At C:\Program Files\WindowsPowerShell\Modules\power-nvm\0.3.0\power-nvm.psm1:187 char:5
+     Move-Item $folder[0].FullName $targetDir;
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : NullArray
```

```
$PSVersionTable.PSVersion

Major  Minor  Build  Revision
-----  -----  -----  --------
5      0      10586  117     
```

Windows 7 SP1

Any clue what can be?

**Comment by brunolm** [comment]
Hmm no idea, I will look into that, could you tell me which power-nvm version do you have installed?

```
Get-Module power-nvm
```

**Comment by fcallejon** [comment]
```
PS C:\Windows\system32> Get-Module power-nvm

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Script     0.3.0      power-nvm                           nvm
```

NODE_DIR returns a value in no-ps terminal (the normal windows command line).
Do I need to re-login/restart? (i'll try it later)

**Comment by brunolm** [comment]
You shouldn't need to restart... well it is setting `NODE_ENV` on User scope, maybe I have to change that to set on machine scope, you could try a manual fix for now with:

```
$x = "C:\Program Files\nodejs";
$env:NODE_DIR = $x;
[Environment]::SetEnvironmentVariable('NODE_DIR', $x, [EnvironmentVariableTarget]::Machine);
````

**Comment by brunolm** [comment]
Oh, no, nevermind, I found the issue, I'm using the wrong variable if you don't have `7z` on path, I will fix and release a version 0.3.1

**Comment by brunolm** [comment]
@fcallejon thank you for reporting this issue, I would've never seen it. I use 7z which is significantly faster than the `Expand-Archive` command. If you add `7z` on your path this module will use that instead.

I believe I fixed this issue now (I was able to reproduce by removing 7z, then I fixed it).

The new version is 0.3.1, if you have any more issues let me know!
